invenio_oauthclient/templates/semantic-ui/invenio_oauthclient/_macros.html

```diff
@@ -9,8 +9,8 @@
 
 {% macro render_checkbox(field, label='') %}
   <div class="ui checkbox">
-    {{field(class_="form-control")}}
-    <label>{{label}}</label>
+    {{ field(class_="form-control") }}
+    {{ field.label }}
```

Comment:
This will render the label as `<label for='...'>text</label>` which will allow it to be clicked on to toggle the checkbox.